cdap-kafka/src/main/java/co/cask/cdap/kafka/run/KafkaServerMain.java

```diff
@@ -92,6 +93,11 @@ public void init(String[] args) {
     if (hostname != null) {
       if (address.isAnyLocalAddress()) {
         kafkaProperties.remove("host.name");
```

Comment:
It was added as a fix for https://issues.cask.co/browse/CDAP-6070 (https://github.com/caskdata/cdap/pull/5888), so that kafka will bind to all interfaces.